src/App.js

```diff
@@ -1,26 +1,700 @@
+                { res: 'Passed', link: '' },
+                { res: 'Passed', link: '' },
+                { res: 'Passed', link: '' },
+                { res: 'Passed', link: '' },
+                { res: 'Passed', link: '' },
+                { res: 'Passed', link: '' },
+                { res: 'Passed', link: '' },
+              ],
+              staging: [
+                { res: 'Passed', link: '' },
+                { res: 'Passed', link: '' },
+                { res: 'Passed', link: '' },
+                { res: 'Passed', link: '' },
+                { res: 'Passed', link: '' },
+                { res: 'Passed', link: '' },
+                { res: 'Passed', link: '' },
+                { res: 'Passed', link: '' },
+                { res: 'Passed', link: '' },
+                { res: 'Passed', link: '' },
+                { res: 'Passed', link: '' },
+                { res: 'Passed', link: '' },
+                { res: 'Passed', link: '' },
+                { res: 'Passed', link: '' },
+              ],
+            },
+          },
+        },
+      },
+    ],
+  };
+
+  searchHandler = (event) => {
+    let val = event.target.value;
+    this.setState({
+      showApp: event.target.value,
+      showTests: 'Void',
+      showEnv: 'Void',
+      showTable: false,
+    });
+  };
+
+  testsSearchHandler = (event) => {
+    let val = event.target.value;
+    this.setState({
+      showTests: event.target.value,
+      showEnv: 'Void',
+      showTable: false,
+    });
+  };
+
+  envSearchHandler = (event) => {
+    let val = event.target.value;
+    this.setState({ showEnv: event.target.value, showTable: true });
+  };
+
+  render() {
+    const style = {
+      backgroundColor: 'green',
+      color: 'white',
+      font: 'inherit',
+      float: 'left',
+      border: '1px solid blue',
+      padding: '8px',
+    };
+
+    let content = [];
+
+    if (
+      this.state.showApp !== 'Void' &&
+      this.state.showEnv !== 'Void ' &&
+      this.state.showTests !== 'Void'
+    ) {
+      for (let i = 0; i < this.state.testResults.length; i++) {
+        content.push(
+          this.state.testResults[i].app[this.state.showApp][
+            this.state.showTests
+          ][this.state.showEnv]
+        );
+      }
+    } else {
+      content = ['Please make your choice!'];
+    }
+
+    let testSelect =
+      this.state.showApp === 'consumer' ? (
+        <div>
+          <p>Test name</p>
+          <select
+            value={this.state.showTests}
+            id="consumerTests"
+            onChange={this.testsSearchHandler}
+          >
+            <option value="Void">...</option>
+            <option value="consumerEmailVerify">Consumer Email Verify</option>
+            <option value="consumerBind">Consumer Bind</option>
+            <option value="consumerPdfVerify">Consumer Pdf Verify</option>
+          </select>
+        </div>
+      ) : this.state.showApp === 'agency' ? (
+        <div>
```

Comment:
This also seems like it should be its own component, or a configurable version of the one above.